### PR TITLE
Match syntax of libc_bitflags! with bitflags!

### PR DESCRIPTION
**Pull request by Susurrus**
Also update a couple of constant declarations while we're at it.

### CODE REVIEW

**Comment by Susurrus** [comment]
@asomers Want to give this a quick glance-over for me? It should be pretty straight-forward.

File: src/sys/select.rs [hunk 1]
```diff
@@ -1,70 +1,38 @@
...
-const BITS: usize = 64;
+pub struct FdSet(libc::fd_set);
 
 impl FdSet {
     pub fn new() -> FdSet {
-        FdSet {
-            bits: [0; FD_SETSIZE as usize / BITS]
-        }
+        let mut fdset = unsafe { mem::uninitialized() };
+        unsafe { libc::FD_ZERO(&mut fdset) };
```
Comment (asomers) [inline src/sys/select.rs]: This shouldn't require `unsafe`.  `FD_ZERO` is implemented in pure Rust.  Same for `FD_CLR`, `FD_SET`, and `FD_ISSET`

Comment (Susurrus) [inline src/sys/select.rs]: Same thing as `WIFCONTINUED`, this is how they're defined in `libc`:
```rust
pub unsafe extern "C" fn FD_SET(fd: c_int, set: *mut fd_set);
```

**Comment by Susurrus** [comment]
@asomers Want to give this another look over now? Should be RFM.

**Comment by asomers** [comment]
LGTM.  But can you squash the last commit?

**Comment by Susurrus** [comment]
@asomers Squash that last commit into which other commit? I left it separate because it didn't seem appropriate to merge it with any other changes in this patchset.

**Comment by asomers** [comment]
Sorry, I thought that the extra `AtFlags` definition was also introduced by this PR.  But now I see that it wasn't.

bors r+

**Pull request merged**